### DISCUSSION TITLE
Vercel Account Registration

### DISCUSSION THREAD

**Discussion by favoursyre** [vercel/community#3904] [Account Issue] [answered]
### Question

I'm trying to sign up with my gmail account but I keep getting an error message whenever I try to log in `We are unable to complete your sign up`

I also tried signing in with my github account but it keeps telling me that the user account has been blocked.

Then I sent a message to registration@vercel.com but I'm still yet to get a reply 

Please can you kindly help me with a way to resolve this issue?

**Comment by Geeyehm** [comment]
Have the same issue, sent an email to that address late last month and still no reply. Also seems like they're ignoring the numerous threads on here complaining about it. Weird.

**Reply by night-slayer18** [reply]
True i guess we'll have to find some other platform that waste time here with people who doesn't want to fix this. They could easily fix it from their side but they ain't doing it

**Comment by leerob** [comment, marked as answer]
Sorry you're having trouble signing up for a new account. We can't safely discuss account information in public because of security concerns. You may have received an email with more details about the error and instructions on next steps. 

For more info, please check this guide: [Why am I unable to login or signup to the Vercel platform?](https://vercel.com/guides/why-can-i-not-signup)
